hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/config/docker-compose.sample.yml

```diff
@@ -27,6 +27,19 @@ services:
       - 8082:8082
 
   Carlos:
+<<<<<<< HEAD
+    container_name: Carlos
+    image: hederahashgraph/hcs-sxc-java-settlement-demo:latest
+    restart: on-failure
+    environment:
+      APP_ID: 2
+      PUBKEY: 302a300506032b6570032100bcabbb31c4c6418ea323e02dd46c060f82936141b5d2d2a1da89e59e1267ab6c
+      ROLES: "BUYER,SELLER"
+      PAYMENT_ACCOUNT_DETAILS: "22-11-14 70925098"
+      COLOR: 275f4f
+    ports:
+      - 8083:8083
+=======
```

Comment:
Merge gone wrong ?

Reply:
I had disabled the examples. merging bits only until things work.